ZelBack/src/services/fluxService.js

```diff
@@ -927,6 +929,13 @@ async function getFluxInfo(req, res) {
       throw nodeJsVersionsRes.data;
     }
     info.flux.nodeJsVersion = nodeJsVersionsRes.data.node;
+    const syncthingVersion = await syncthingService.systemVersion();
+    if (syncthingVersion.status === 'error') {
+      throw syncthingVersion.data;
+    }
+    info.flux.syncthingVersion = syncthingVersion.data.version;
```

Comment:
this is causing tests to fail as the syncthing dir hasn't been updated and getting EACCESS. Probably needs to be stubbed